filebeat/docs/fields.asciidoc

```diff
@@ -1309,6 +1309,33 @@ type: keyword
 --
 Identifies the VPC endpoint in which requests were made from a VPC to another AWS service, such as Amazon S3.
 
+type: keyword
+
+--
+
+[float]
+=== cloudwatch
+
+Fields for AWS CloudWatch logs.
+
+
+
+*`aws.cloudwatch.ip`*::
++
+--
+The internet address of the requester.
+
+
+type: ip
+
+--
+
+*`aws.cloudwatch.program_name`*::
```

Comment:
Is it the log group? If so, maybe keep the original naming?

Reply:
It's not log group 😬 It is `systemd` part in `2020-02-20T07:01:01.000Z Feb 20 07:01:01 ip-172-31-81-156 systemd: Stopping User Slice of root`.

Reply:
I see... hmm, that suggests "service_name", "application_name", but I'm fine with program too :)

Reply:
I think I got this program_name from some AWS CLI. If I find it, I will link it here 😬

Reply:
I decide to change it to process.name and process.pid to match what system filebeat module has.